### PR TITLE
[update] MIP-0 : add codeowners

### PR DESCRIPTION
**Pull request by apenzk**
## Summary

Codeowners should get notified if there are relevant changes to the repo.

## Changes

- Following the following advice I added a couple people as codeowners to get notified on new PRs. 

> ... you can raise a PR including a `CODEOWNERS` in .github/ 

- In addition, authors should add themselves to `CODEOWNERS` to get notified, if there is a change to their MIP/MD.

### CODE REVIEW

**Comment by apenzk** [comment]
@andyjsbell i added you and l-monninger. anyone else?

**Comment by andyjsbell** [comment]
I guess that can do for the time 👍

**Review by andyjsbell — APPROVED** [review]
thanks @apenzk

**Review by franck44 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Reviewers that should be added to all additions
```
Comment (apenzk) [inline .github/CODEOWNERS:1]:
```suggestion
# Reviewers that should be generally notified.
```

**Review by l-monninger — APPROVED** [review]

**Pull request merged**